topics/transcriptomics/tutorials/scrna-umis/tutorial.md

```diff
@@ -0,0 +1,553 @@
+
+
+Here we can see the three distinct regions along the x-axis that correspond to our expected [CEL-Seq2 Schema](#the-cel-seq2-protocol):
+
+ * 01-06bp: smooth, relatively constant bases.
+ * 07-12bp: noisy, highly varied distribution of bases.
+ * 13-30bp: T-dominated region
+
+We can see that the distribution of the first 6bp is relatively more even than the following 6bp which seems to have more extreme variation.
+
+> ### {% icon question %} Question
+>
+> Why is this the case? Why is the UMI barcode base distribution smoother than the Cell barcode base distribution?
+> >
+> > ### {% icon solution %} Solution
+> >
+> > There are far more UMIs than cells. Cell barcodes are designed and selected with a specified edit distance, greatly limiting their availability in the data. UMIs are not so well-curated -- i.e it is possible to encounter the same UMI in the same cell multiple times. The more extreme variation in the 7-12bp region is simply caused by a fewer number of samples.
+> >
+> {: .solution}
+{: .question}
+
+
+# Uniting Barcodes with Sequence
+
+In a sense, we have a disparity in our data: the reverse reads contain the sequences we wish to map, but not the barcodes; the forward reads contain the barcode, but not the sequence. For the forward and reverse reads given above, the information that we really want from both can be summarized in this table:
+
+ | Read | Cell | UMI | Sequence |
+ |------|------|-----|----------|
+ | <small>@J00182:75:HTKJNBBXX:2:1115:8501:46961</small>  | ACCAGA | GGAAGA | <small>GACCTCTGATCTTTACGAAAGGCCAACGCGTTTTCAGTCTGGACACGGTTCAGCTCCTGTTCATTATTCA</small> |
+ | <small>@J00182:75:HTKJNBBXX:2:1203:25022:13763</small> | GGTAAC | GTCCCA | <small>GCCACCTAATTTCCGTCATCACACTCCTCTCCGTTTTCAACTTGCACAATGCTGTCTCCGCAGAATCCCT</small> |
+ | <small>@J00182:75:HTKJNBBXX:2:2222:13301:35690</small> | GGTAAC | GTCCCA | <small>CAATCCTCTCCGTTATCAACTTGCACAATGCTGTCTCCGCAGAATCCCTCCGGATCAGGATCGCTCTCCA</small> |
+ | <small>@J00182:75:HTKJNBBXX:2:1114:12469:11073</small> | GGTAAC | CGGCGT | <small>ATCCACTTATTGCAAAGCAGAGGACATTGAGTCTCACCTTTTGTCCAGGTCTTCCAATTTCACCCTGCAA</small> |
+
+> ### {% icon question %} Question
+>
+> Provided that these reads all map to the same gene:
+>
+> 1. Which of these reads come from the same cell?
+> 2. Which of these reads are duplicates?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. Reads:
+> >  * `@J00182:75:HTKJNBBXX:2:1203:25022:13763`
+> >  * `@J00182:75:HTKJNBBXX:2:2222:13301:35690`
+> >  * `@J00182:75:HTKJNBBXX:2:1114:12469:11073`
+> >  all have the cell barcode `GGTAAC`.
+> >
+> > 2. Reads:
+> >  * `@J00182:75:HTKJNBBXX:2:1203:25022:13763`
+> >  * `@J00182:75:HTKJNBBXX:2:2222:13301:35690`
+> >  are duplicates, since they both have the same cell barcode and same UMI.
+> >
+> > However, they are not direct duplicates of each other. since their sequences are different. However, they do stem from the same read as evidenced by their overlap:
```

Comment:
other, since?

Reply:
What does this mean if you don't have a direct duplicate? Does it get filtered? Does it matter?